node/task.ts

```diff
@@ -1116,6 +1204,20 @@ export function _loadData(): void {
         }
     }
     debug('loaded ' + loaded);
+
+    // store public variable metadata
+    JSON.parse(process.env['VSTS_PUBLIC_VARIABLES'] || '[]')
```

Comment:
I'll wrap in try catch for the interactive dev scenario - to gives the developer additional context what is blowing up. No need to loc the message.